images/salt-master/Dockerfile

```diff
@@ -4,14 +4,6 @@ MAINTAINER moonshot-platform <moonshot-platform@scality.com>
 
 # Versions to use
 ARG SALT_VERSION=2018.3.3
-ARG TINI_VERSION=v0.18.0
-
-# Install and check tini
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /tini.asc
-
-COPY source.key source.key
```

Comment:
Would be nice to untrack this file as well ;)

Reply:
Will do